src/pymatgen/analysis/phase_diagram.py

```diff
@@ -2781,18 +2781,36 @@ def _create_plotly_fill(self):
                     ys.append(y)
                     zs.append(z)
 
-                traces += [
-                    go.Mesh3d(
-                        x=xs,
-                        y=ys,
-                        z=zs,
-                        opacity=0.05,
-                        alphahull=-1,
-                        flatshading=True,
-                        hoverinfo="skip",
-                        color=next(fillcolors),
-                    )
-                ]
+                if _idx == 1:
+                    traces += [
+                        go.Mesh3d(
+                            x=xs,
+                            y=ys,
+                            z=zs,
+                            opacity=0.05,
+                            alphahull=-1,
+                            flatshading=True,
+                            hoverinfo="skip",
+                            color=next(fillcolors),
+                            legendgroup="facets",
+                            showlegend=True,
+                            name="Hull Surfaces (toggle to access points easier)",
+                        )
+                    ]
+                else:
+                    traces += [
+                        go.Mesh3d(
+                            x=xs,
+                            y=ys,
+                            z=zs,
+                            opacity=0.05,
+                            alphahull=-1,
+                            flatshading=True,
+                            hoverinfo="skip",
+                            color=next(fillcolors),
+                            legendgroup="facets",
+                        )
+                    ]
```

Comment:
Adding the legendgroup to the facets of the quaternary graph. The first facet is used as the entry to the legend and defines the name which appears in the legend.